HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.milab_hw2;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+
+public class StarksGalleryActivity extends GalleryActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_starks_gallery);
+
+        String[] starks_data= {"Arya", "And such"};
```

Comment:
Don't mind if you had only 1 stark family member but please complete the implementation.

Reply:
I re-uploaded the files (as I wrote this was an older version of my code), please check it out now :) Sorry for the inconvenience